### PR TITLE
Update kakarot links

### PR DESCRIPTION
**Pull request by pucedoteth**
i have updated the contributors link and make clickable also added Kakarot official discord.

Time spent on this PR: 0.5 day

## Pull request type

Please check the type of change your PR introduces:

- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

you can find the official Kakarot discord and when you click on Contributors it takes you to the right page.

Resolves #<Issue number>

## What is the new behavior?

-you can find the official Kakarot discord.
-when you click on Contributors it takes you to the right page.

- - -
This change is [Reviewable](https://reviewable.io/reviews/kkrt-labs/kakarot/884)

### CODE REVIEW

**Review by ClementWalter — CHANGES_REQUESTED** [review]
fix repo base address in github

File: README.md [hunk 1]
```diff
@@ -10,12 +10,13 @@
 
 ![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/kkrt-labs/kakarot/ci.yml?branch=main)
 ![GitHub](https://img.shields.io/github/license/abdelhamidbakhta/kakarot?style=flat-square&logo=github)
-![GitHub contributors](https://img.shields.io/github/contributors/abdelhamidbakhta/kakarot?logo=github&style=flat-square)
+[![GitHub contributors](https://img.shields.io/github/contributors/abdelhamidbakhta/kakarot?logo=github&style=flat-square)](https://github.com/abdelhamidbakhta/kakarot/graphs/contributors)
```
[the remark this answers is not in the document]
Comment (pucedoteth) [reply README.md]: > repo is at kkrt-labs now and not abdelhamidbakhta

fixed

**Review by ClementWalter — CHANGES_REQUESTED** [review]
since you're making these changes, can you update all the links using abdelhamid repo instead of this one?

**Comment by pucedoteth** [comment]
> fix repo base address in github

i have updated links as you requested. removed abdelhamid and insert kkrt-labs

**Review by ClementWalter — APPROVED** [review]

**Pull request merged**